pytd/writer.py

```diff
@@ -437,6 +438,8 @@ def write_dataframe(self, dataframe, table, if_exists, fmt="csv", keep_list=Fals
         self._bulk_import(table, fp, if_exists, fmt)
 
         fp.close()
+        if fmt == "csv":
```

Comment:
To ensure deleting temporary when pytd gets an exception, how about using `contextlib.ExitStack()` or `contextlib.contextmanager`?
https://docs.python.org/3/library/contextlib.html#replacing-any-use-of-try-finally-and-flag-variables
https://docs.python.org/ja/3/library/contextlib.html#contextlib.contextmanager

Reply:
Nice catch! How about this ec12cb8e9b8874dfdfe2fccaf71f6a580bab8944 with a new test case d02a158783f6258bdc1327db0a7c743e1086ae5a?

Reply:
Or, we may try to use `BytesIO` instead of writing temporary file.